independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json

```diff
@@ -49,23 +54,60 @@
             "x-ms-summary": "Search criteria",
             "x-ms-visibility": "advanced"
           },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Filters results (rows)",
+            "x-ms-summary": "Filter criteria",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "ConsistencyLevel",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "Set this property to 'eventual' if you are using $search",
+            "x-ms-summary": "Enable advanced queries",
+            "x-ms-visibility": "advanced",
+			"enum": [
+              "eventual",
+              ""

```

Comment:
@paulculmsee is it possible to remove the empty string?

Reply:
There is only 1 value for that property - it is there or it is not. I could not see a better way to do it. Thoughts @Amjed-Ayoub ?

Reply:
Removed property entirely and went with a policy template